src/main/java/com/bullhornsdk/data/model/entity/core/standard/PlacementTimeAndExpense.java

```diff
@@ -119,6 +122,10 @@ public class PlacementTimeAndExpense extends AbstractEntity implements QueryEnti
     @Size(max = 32)
     private String vmsRequisitionID;
 
+    @Getter
+    @Setter
+    private DateTime timeAndLaborEnabledDate;
```

Comment:
@scriptom 
Can you put this property in the @JsonPropertyOrder annotation for consistency?